### PR TITLE
Feature/form styling

### PR DESCRIPTION
**Pull request by teenie-quaggard**
### NOTE: Only merge this branch after PR #158 is merged. 

What's been done since @teenie-quaggard last pushed:

- [x] `Location` has been turn into a dropdown instead of a textField. related to issue #149 

- [x] `Additional comments` field has been added. 

- [x] `data.json` has been updated with the locations provided. 

- [x] update text-inputs to use highlight color for line and label (and radio buttons). related to issue #141.

- [x]  allow for 'details' & connection to wlm to be page-width input

- [x] Form has been styled in accordance to Figma design. #130 

- [x]  merged this branch with branch `feature/sendToBe` to style the buttons. 

- [x] Made `DatePicker` more accessible and user-friendly. 

Still needs (up for discussion still):
- instead of dropdown for category / tags, it might actually make more sense to lay them all visible? what do you think (category is radio, tags is checkboxes)

### CODE REVIEW

**Comment by orjwan98** [comment]
@teenie-quaggard I agree with making the Tags into checkboxes but I think categories should stay as it is. Especially if they decide to add more categories it would make it more messy and less visually appealing. What do you think?

**Comment by teenie-quaggard** [comment]
@orjwan98 looking really good so far! looks like the form is nearly done!
small error in the browser to look at:
Screen Shot 2019-05-19 at 23 10 47

**Comment by orjwan98** [comment]
@teenie-quaggard fixed the errors and it's ready for review :)

**Comment by teenie-quaggard** [comment]
On issues that I have is that when I'm tabbing through the site, I cannot change the date, which is an accessibility problem. 

Also - if I click enter, anywhere on the form it totally submits the entire form!

Atm, this color combo doesn't have enough contrast.
Screen Shot 2019-05-20 at 11 17 19
Can we darken the orange, or make the text black?

If I type in the connection to WLM box, my input is copied into the additional comments box. Related to that, I cannot edit the comments box.
Screen Shot 2019-05-20 at 11 19 55

Small thing - but I think that the image upload should come before the input for the image caption.

BTW!! You've killed it on this form!! For real - we are nearly there!!! 🎉

**Comment by orjwan98** [comment]
@teenie-quaggard what do you mean you can't change the date? :/

**Comment by teenie-quaggard** [comment]
> @teenie-quaggard what do you mean you can't change the date? :/

@orjwan98 if you pretend that you can't use the mouse and are navigating the website by using tab you should be able to tab down the form and control all inputs using only your keyboard. i seem to be able to access everything else, except for the date!

**Comment by orjwan98** [comment]
@teenie-quaggard I just pushed up again, feel free to look at it again :)

File: client/src/Components/Form/AdditionalInfo/muiStyles.js [hunk 1]
```diff
@@ -14,7 +40,20 @@ export const styles = theme => ({
     overflowY: 'auto',
   },
   chip: {
-    margin: theme.spacing / 4,
+    background: 'rgba(192, 39, 0, 0.7)',
+    margin: '3px',
```
[the remark this answers is not in the document]
Comment (orjwan98) [reply client/src/Components/Form/AdditionalInfo/muiStyles.js]: Fixed :D

**Comment by orjwan98** [comment]
@teenie-quaggard @Oliversw  @mhmdtshref  I just pushed up again with conflicts resolved. Feel free to look at it again :)

**Comment by Oliversw** [comment]
@orjwan98 @teenie-quaggard Looks so much better, thank you! The only thing that I encountered was this error (screen-shotted below) whenever I set a value in the date picker. It seems that the date picker is classified as an 'uncontrolled component' whereas we use 'controlled components' everywhere else. 

From the React docs:
> In a controlled component, form data is handled by a React component. The alternative is uncontrolled components, where form data is handled by the DOM itself.

The error seems to occur when you select a date, which then modifies the state, making it a controlled component. It seems to be easily fixed though! If we just include a default date, which is already predefined in the state on page load, there is no longer an error, since it starts as a controlled component. 😄 
E.g:
```    
   selectedDate: 'May 2019',
    year: '2019',
    month: 'May',
```
The error:
Screen Shot 2019-05-20 at 12 26 16 PM

**Comment by orjwan98** [comment]
@Oliversw all is fixed :) review one last time?

**Pull request merged**